CONTRIBUTING

```diff
@@ -1,26 +1,26 @@
-HOW TO CONTRIBUTE PATCHES TO OpenSSL
-------------------------------------
+HOW TO CONTRIBUTE TO OpenSSL
+----------------------------
 
 (Please visit https://www.openssl.org/community/getting-started.html for
 other ideas about how to contribute.)
 
-Development is coordinated on the openssl-dev mailing list (see the
-above link or https://mta.openssl.org for information on subscribing).
-If you are unsure as to whether a feature will be useful for the general
-OpenSSL community you might want to discuss it on the openssl-dev mailing
-list first.  Someone may be already working on the same thing or there
-may be a good reason as to why that feature isn't implemented.
+Development is done on GitHub, https://github.com/openssl/openssl.
 
-To submit a patch, make a pull request on GitHub.  If you think the patch
-could use feedback from the community, please start a thread on openssl-dev
-to discuss it.
+To request new features or report bugs, please open an issue.
 
-Having addressed the following items before the PR will help make the
-acceptance and review process faster:
+To submit a patch, make a pull request on GitHub.  If you are thinking of
```

Comment:
"To submit a patch, please open a pull request on GitHub..."

Reply:
There is some steps before opening a PR that can be mentionned, like cloning the OpenSSL repository, create our work branch, push the change onto it, and them create a PR with this edited branch.

Reply:
I took the wording change, but this is not the place for a GitHub tutorial, so not going to do that.  Having that kind of doc, probably on our website, makes sense.